add_feature_info --- unknown to Cmake on Mac

Hello.  I followed the Mac OS instructions, installed all the dependencies, and I am not able to build.  Originally the instructions said to make a build direction and build from there, but it couldn't find Cmake lists.  So I build from src.  Any thoughts on the error?

My Cmake options are 
-DCMAKE_INSTALL_PREFIX=/users/Me/documents/ -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release -DWITH_XC_AUTOTYPE=ON -DCMAKE_PREFIX_PATH=/usr/local/cellar/qt/5.12.2/lib/cmake/

CMake Error at CMakeLists.txt:191 (add_feature_info):
  Unknown CMake command "add_feature_info".

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,6 +36,7 @@ endif (CCACHE_FOUND)
 
 # Support Visual Studio Code
 include(CMakeToolsHelpers OPTIONAL)
+include(FeatureSummary)
 
 include(CheckCCompilerFlag)
 include(CheckCXXCompilerFlag)
@@ -464,8 +465,6 @@ if(UNIX)
 endif()
 
 include_directories(SYSTEM ${GCRYPT_INCLUDE_DIR} ${ZLIB_INCLUDE_DIR})
-
-include(FeatureSummary)
 
 add_subdirectory(src)
 add_subdirectory(share)

